superset-frontend/tsconfig.json

```diff
@@ -3,10 +3,10 @@
     "allowJs": true,
     "allowSyntheticDefaultImports": true,
     "baseUrl": ".",
-    "esModuleInterop": true,
+    "esModuleInterop": false,
     "forceConsistentCasingInFileNames": true,
-    "importHelpers": true,
-    "jsx": "react",
+    "importHelpers": false,
+    "jsx": "preserve",
```

Comment:
`.tsx?` files will now pass through `babel-loader` too, so let Babel handle `JSX` as recommended in https://github.com/TypeStrong/ts-loader#appendtsxsuffixto